project/src/main/webapp/details-page/details_script.js

```diff
@@ -0,0 +1,188 @@
+var latitude;
+var longitude;
+var map;
+
+function GetURLParameter(sParam) {
+    var sPageURL = window.location.search.substring(1);
+    var sURLVariables = sPageURL.split('&');
+    for (var i = 0; i < sURLVariables.length; i++) {
+        var sParameterName = sURLVariables[i].split('=');
+        if (sParameterName[0] == sParam) {
+            return sParameterName[1];
+        }
+    }
+}
+
+
+async function loadDetails() {
+    const entryID = GetURLParameter("entryID");
+
+    const fetchedJSON = await fetch("/get-by-id?entryID=" + entryID);
+    const report = await fetchedJSON.json();
+
+    document.getElementById("title-container").innerText = report.title;
+    document.getElementById("date-container").innerText = new Date(report.date).toDateString();
+    document.getElementById("description-container").innerText = report.description;
+    document.getElementById("contact-details-container").innerText = report.contactDetails;
+    document.getElementById("image-container").src = report.imageURL;
+
+    document.getElementById("entry-id-container").value = entryID;
+
+    latitude = parseInt(report.latitude);
+    longitude = parseInt(report.longitude);
+
+}
+
+
+function createMap() {
+    // Create a new StyledMapType object, passing it an array of styles,
+    // and the name to be displayed on the map type control.
+    const styledMapType = new google.maps.StyledMapType(
```

Comment:
Is this the same style that Chloe is using? If it is, then it might make sense to put it in a library that the two of you can call.